doc/script_commands.txt

```diff
@@ -5389,7 +5389,8 @@ and theirs val1, val2, val3, and val4 usage in source.
 perform a complete removal of all statuses (although permanent ones will re-apply).
 
 'sc_end_class' works like 'sc_end' but will remove all status effects from any learned
-skill on the invoking character.
+skill on the invoking character. if <job_id> was given it will end the effect for that job.
+if the <char_id> was -1 it will return the <char_id> for the invoking character
```

Comment:
`sc_end_class` + `job_id` will be used in few case. I believe adding a shortcut for attached char ID is meaningless when `getcharid(3)` does the job.

Reply:
true when i did that i was thinking about getpartymember i forgot that even if can be -1 it will eventually needed to use $@partymembercid[]
i will remove the -1 option